### ISSUE TITLE
[Bug?] Extended reach isn't applying?

### ISSUE THREAD

**Issue by Akitori** [coolAlias/DynamicSwordSkills#12]
So i have a small mod that lets me extend reach using forge's method. However, i can't seem to hit any enemies further than usual? Is that intentional?

**Comment by coolAlias** [comment]
It's not intentional - I didn't realize Forge (or Minecraft?) had added a REACH attribute.

However, as far as I can tell, for the purpose of attacking entities, reach is still hard-coded in Minecraft to be limited to 6 blocks' distance when in Creative and 3 blocks' distance otherwise. See the code for `EntityRenderer#getMouseOver`. This is also enforced server-side in `NetHandlerPlayServer#processUseEntity`.

In other words, it looks to me like the REACH attribute is solely intended for interacting with blocks and has no bearing on the distance at which you can attack entities, in which case DSS is working as intended.

Do you have a link to the source code of the mod so I can see what they're doing? I suspect there won't be any way to make them compatible, but if there is, the source code will help me find it.

**Comment by Akitori** [comment]
Well my friend (tmtravlr) is working on a pull request to get that feature added to forge itself. As they don't have a github of potioncore updated, perhaps the pull request would give some hint? (i will however remind tm of updating their github soon enough)

https://github.com/MinecraftForge/MinecraftForge/pull/4736

**Comment by coolAlias** [comment]
Looks like I'll have to wait for that PR to make it into Forge before this can be resolved on my end.

**Comment by coolAlias** [comment]
Added the wontfix label for now since the PR is stale and may never be incorporated. Will reopen if ever addressed in Forge/Minecraft.
